testsuite/tests/singlecluster/extensions/telemetry_policy/conftest.py

```diff
@@ -0,0 +1,71 @@
+"""Conftest for TelemetryPolicy tests"""
+
+import pytest
+
+from testsuite.kuadrant.extensions.telemetry_policy import TelemetryPolicy
+from testsuite.kuadrant.policy.rate_limit import Limit
+from testsuite.kuadrant.policy import CelPredicate
+from testsuite.kubernetes.monitoring import MetricsEndpoint
+from testsuite.kubernetes.monitoring.service_monitor import ServiceMonitor
+
+
+@pytest.fixture(scope="session")
+def limitador(kuadrant):
+    """Returns Limitador CR"""
+    return kuadrant.limitador
```

Comment:
A bit useless fixture that deep inside the testsuite. Access name straight through `kuadrant.limitador.name()` if you need it only once